### PR TITLE
fix(web-app): dropdown border styles

### LINKED ISSUE #1125: Dropdown style overrides need fixing
**Description**

I'm seeing broken dropdown overrides in the assets table:
image

It should look like this:
image

Same with catalog sort:
image

It should look like this:
image

This could be happening in other places too; I didn't exhaustively search for all dropdowns in the web app.

**Steps to reproduce**

I'm not seeing this on production. I'm seeing it on localhost.

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #1125

Dropdown override border style fix

#### Testing / reviewing

Check that dropdowns look correct
http://localhost:3000/libraries/carbon-charts/latest/assets
http://localhost:3000/assets/components

### CODE REVIEW

File: services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/assets/index.module.scss [hunk 1]
```diff
@@ -59,6 +59,7 @@
   display: flex;
   width: 100%;
   padding-left: spacing.$spacing-05;
+  border-bottom: 1px solid theme.$border-subtle;
```
Comment (francinelucca) [inline services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/assets/index.module.scss:62]: is this one necessary I see a weird mismatch now in the gray part (see to the right, where it ends)
image

**Review by francinelucca — CHANGES_REQUESTED** [review]
looks good on catalog, minor visual comment on library assets

File: services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/assets/index.module.scss (hunk 1 above)
Comment (alisonjoseph) [reply services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/assets/index.module.scss:62]: Good catch, removed!

**Review by andreancardona — CHANGES_REQUESTED** [review]
on focus it's missing `1px` in the bottom border 
Screen Shot 2022-08-16 at 11 55 14 AM

**Comment by alisonjoseph** [comment]
@andreancardona good catch! fixed

**Review by andreancardona — APPROVED** [review]

**Review by francinelucca — APPROVED** [review]
✨

**Pull request merged**